extensions/amp-yotpo/validator-amp-yotpo.protoascii

```diff
@@ -0,0 +1,65 @@
+#
+# Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-yotpo
+  html_format: AMP
+  tag_name: "SCRIPT"
+  spec_name: "amp-yotpo extension .js script"
+  satisfies: "amp-yotpo extension .js script"
+  requires: "amp-yotpo"
+  mandatory_parent: "HEAD"
+  unique: true
+  extension_unused_unless_tag_present: "amp-yotpo"
+  attrs: {
+    name: "async"
+    mandatory: true
+    value: ""
+  }
+  attrs: {
+    name: "custom-element"
+    mandatory: true
+    value: "amp-yotpo"
+    dispatch_key: true
+  }
+  attrs: { name: "nonce" }
+  attrs: {
+    name: "src"
+    mandatory: true
+    value_regex: "https://cdn\.ampproject\.org/v0/amp-yotpo-(latest|0\.1).js"
+  }
+  attrs: {
+    name: "type"
+    value: "text/javascript"
+  }
+  cdata: {
+    blacklisted_cdata_regex: {
+      regex: "."
+      error_message: "contents"
+    }
+  }
+  spec_url: "https://www.ampproject.org/docs/reference/components/amp-yotpo"
+}
+tags: {  # <amp-yotpo>
```

Comment:
please add the mandatory attributes here in the form of 
```
  attrs: {
    name: "data-foo"
    mandatory: true
  }
```

Reply:
Done